<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>672</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="scalingLayout">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Scaling</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_14">
      <item>
       <widget class="QLabel" name="scalingLabel">
        <property name="text">
         <string>Just pan (zoom) the plot with your mouse (wheel). Click &quot;Reset View&quot; button for automatic scaling. Right click on plot to set up fixed scaling.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="graphGroupBox">
     <property name="title">
      <string>Graph</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="plotLineWidthLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Line width</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="CustomDoubleSpinBoxNoSign" name="plotLineWidthSpinBox">
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="minimum">
           <double>0.100000000000000</double>
          </property>
          <property name="maximum">
           <double>10.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>2.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="plotLineOpacityLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Line opacity</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="CustomSpinBox" name="plotLineOpacitySpinBox">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="value">
           <number>200</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="fillOpacityLayout">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Fill opacity</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="CustomSpinBox" name="plotFillOpacitySpinBox">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="value">
           <number>200</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="displayColorLayoutErrorCombined">
        <item>
         <widget class="QLabel" name="colorLabelErrorCombined">
          <property name="text">
           <string>(Combined) Error Signal</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="colorPreviewLabelErrorCombined">
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="editColorButtonErrorCombined">
          <property name="text">
           <string>...</string>
          </property>
          <property name="arrowType">
           <enum>Qt::NoArrow</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="displayColorLayoutError1">
        <item>
         <widget class="QLabel" name="colorLabelError1">
          <property name="text">
           <string>Error Signal 1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="colorPreviewLabelError1">
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="editColorButtonError1">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="displayColorLayoutError2">
        <item>
         <widget class="QLabel" name="colorLabelError2">
          <property name="text">
           <string>Error Signal 2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="colorPreviewLabelError2">
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="editColorButtonError2">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="displayColorLayoutMonitor">
        <item>
         <widget class="QLabel" name="colorLabelMonitor">
          <property name="text">
           <string>Monitor Signal</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="colorPreviewLabelMonitor">
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="editColorButtonMonitor">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="displayColorLayoutMonitorHistory">
        <item>
         <widget class="QLabel" name="colorLabelMonitorHistory">
          <property name="text">
           <string>Monitor Signal History</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="colorPreviewLabelMonitorHistory">
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="editColorButtonMonitorHistory">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="displayColorLayoutControl">
        <item>
         <widget class="QLabel" name="colorLabelControl">
          <property name="text">
           <string>Control Signal</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="colorPreviewLabelControl">
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="editColorButtonControl">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="displayColorLayoutControlHistory">
        <item>
         <widget class="QLabel" name="colorLabelControlHistory">
          <property name="text">
           <string>Control Signal History</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="colorPreviewLabelControlHistory">
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="editColorButtonControlHistory">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="displayColorLayoutSlowControl">
        <item>
         <widget class="QLabel" name="colorLabelSlowControl">
          <property name="text">
           <string>Slow Control History</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="colorPreviewLabelSlowControl">
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="editColorButtonSlowControl">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_12">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Export data</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_18">
      <item>
       <widget class="QPushButton" name="exportSelectFilePushButton">
        <property name="text">
         <string>Select file</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>no files are overridden. Instead, a suffix is appended.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exportDataPushButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Export</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomDoubleSpinBoxNoSign</class>
   <extends>QDoubleSpinBox</extends>
   <header location="global">spin_box.h</header>
  </customwidget>
  <customwidget>
   <class>CustomSpinBox</class>
   <extends>QSpinBox</extends>
   <header location="global">spin_box.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
